dialogue-core/src/main/java/com/palantir/dialogue/core/DialogueChannel.java

```diff
@@ -97,13 +98,17 @@ public void updateUris(Collection<String> uris) {
         Sets.SetView<String> newUris = Sets.difference(uniqueUris, limitedChannelByUri.keySet());
 
         staleUris.forEach(limitedChannelByUri::remove);
-        newUris.forEach(uri -> limitedChannelByUri.put(uri, createLimitedChannel(uri)));
+        ImmutableList<String> allUris = ImmutableList.<String>builder()
```

Comment:
These indices aren't stable across updates so it will probably pretty confusing to look at individual metrics. Not sure of a better solution, but worth flagging

Reply:
Yep. Given that it's not recommended to just hash things, I think this is the best we can do.  In practise, I hope it'll be reasonably easy to see n coloured lines go into a live-reload event and then either n+1 or n-1 lines come out.  People should be able to visually connect em up.